ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -1,4 +1,5 @@
-import React, { useContext, useState } from 'react';
```

Comment:
I'm wondering if we could remove the `.component` filename post-fix in these new components, since @danjm mentioned that this naming convention is discouraged for all new components? https://github.com/MetaMask/metamask-extension/pull/11404/files#r661634934

Reply:
We can do that in a another step with these components at a different time.

Reply:
Sure, sounds good!